pgctl/cli.py

```diff
@@ -429,7 +429,12 @@ def start(self):
         return self.__show_failure('start', failed)
 
     def stop(self, with_log_running=False):
-        """Idempotent stop of a service or group of services"""
+        """Idempotent stop of a service or group of services
```

Comment:
total nit, but PEP257 suggests to leave a blank line after the summary line in multi-line docstrings: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings